src/App.tsx

```diff
@@ -51,16 +58,21 @@ function App() {
       <ErrorBoundary FallbackComponent={ErrorFallback} onError={ErrorHandler}>
         {redirect && <Redirect to={redirect}></Redirect>}
         {/*  <React.StrictMode>*/}
-        <FeatureToggleProvider featureToggles={{'SURVEY BUILDER': true}}>
+        <FeatureToggleProvider featureToggles={{'SURVEY BUILDER': true, 'USERNAME PASSWORD LOGIN': false}}>
```

Comment:
Set feature toggle for login with username/password to false

Reply:
Is this how the feature is hidden? Do we have a variable in the model for whether or not this is staging? Would be nice to show it on staging but hide on production.

Reply:
Yes, this is how features are hidden right now. There's an `isDevelopment` method that could be used to show locally / on staging, but hide on prod -- I'll update in the next commit. I also opened [a ticket](https://sagebionetworks.jira.com/browse/DHP-1065) to track setting up a button to allow the user to toggle the features, if desired.